src/compiler.js

```diff
@@ -121,6 +121,10 @@ var Compiler = Object.extend({
         return 't_' + this.lastId;
     },
 
+    _templateName: function() {
+        return this.templateName == null? "undefined" : '"'+this.templateName.replace(/"/g, '\\"')+'"';
```

Comment:
Filename can contains quotes (at least on OS X).

I can remove this, if you believe this is not necessary.